Cargo.toml

```diff
@@ -44,9 +44,9 @@ marker_utils   = { path = "./marker_utils", version = "0.5.0-dev" }
 bumpalo            = "3.14"
 camino             = { version = "1.1", features = ["serde1"] }
 cargo_metadata     = "0.18"
-clap               = { version = "=4.3.24", features = ["string", "derive"] }
```

Comment:
I wonder why it was pinned

Reply:
Clap increased the minimal rust version in newer releases. The pin was me trying to keep rust 1.66 as the minimal version. Now we can just use the newest version again.